web/src/search/input/interactive/SelectedFiltersRow.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import FilterInput from './FilterInput'
+import { QueryState } from '../../helpers'
+import { FiltersToTypeAndValue } from '../../../../../shared/src/search/interactive/util'
+
+interface Props {
+    /**
+     * The filters currently added to the query.
+     */
+    filtersInQuery: FiltersToTypeAndValue
+
+    /**
+     * The query in the main query input.
+     */
+    navbarQuery: QueryState
+
+    /**
+     * Callback to trigger a search when a filter is submitted.
+     */
+    onSubmit: (e: React.FormEvent<HTMLFormElement>) => void
+    /**
+     * Callback to handle a filter's value being updated.
+     */
+    onFilterEdited: (filterKey: string, value: string) => void
+
+    /**
+     * Callback to handle a filter being deleted from the selected filter row.
+     */
+    onFilterDeleted: (filterKey: string) => void
+
+    /**
+     * Callback to handle the editable state of a filter.
+     */
+    toggleFilterEditable: (filterKey: string) => void
+
+    /**
+     * Whether we're on the search homepage.
+     */
+    isHomepage: boolean
+}
+
+/**
+ * The row displaying the filters that have been added to the query in interactive mode.
+ */
+export const SelectedFiltersRow: React.FunctionComponent<Props> = ({
+    filtersInQuery,
+    navbarQuery,
+    onSubmit,
+    onFilterEdited,
+    onFilterDeleted,
+    toggleFilterEditable,
+    isHomepage,
+}) => {
+    const filterKeys = Array.from(Object.keys(filtersInQuery))
```

Comment:
```suggestion
    const filterKeys = Object.keys(filtersInQuery)
```
`Object.keys()` returns an array.